### ISSUE TITLE
Syntax Highlighting

### ISSUE THREAD

**Issue by ghost** [Calysto/scilab_kernel#7]
Hello.

I have installed the kernel in Jupyter and it is working fine but I am not sure why the syntax highlighting isn't working. I hope this is the right place to report this as this is my first time on GitHub.

Best wishes.

**Comment by blink1073** [comment]
Hi @NumSim3070, thanks for the report.  I am not sure why the syntax highlighting isn't working.  I'll leave this issue open for when I have a chance to look into it.

**Comment by EmericT** [comment]
Same here. I can reproduce with a docker image :
FROM jupyter/minimal-notebook:latest
USER root
RUN apt-get update && apt-get -y install -y \
  openjdk-8-jre \
  openjdk-8-jre-headless \
  scilab
ARG JUPYTERHUB_VERSION=0.9.4
RUN pip install --no-cache \
    jupyterhub==$JUPYTERHUB_VERSION scilab_kernel
USER jovyan 

If it helps, I get the log line : 
[W 20:38:00.975 NotebookApp] 404 GET /static/components/codemirror/mode/scilab/scilab.js?v=20190313203735 (172.17.0.1) 1.55ms referer=http://127.0.0.1:8888/notebooks/Untitled.ipynb?kernel_name=scilab

**Comment by EmericT** [comment]
Ok, found a solution : I used 
sed -i "s/'name': 'scilab'/'name': 'octave'/" 
on kernel.py

This makes codemirror use octave's highlighting and it works fine.

**Comment by mutahiwachira** [comment]
> Ok, found a solution : I used
> sed -i "s/'name': 'scilab'/'name': 'octave'/"
> on kernel.py
> 
> This makes codemirror use octave's highlighting and it works fine.

I tried to copy in "s/'name': 'scilab'/'name': 'octave'/" below the mimetex line on kernel.py but it didn't work. Can you give some more detail?

**Comment by mutahiwachira** [comment]
> Ok, found a solution : I used
> sed -i "s/'name': 'scilab'/'name': 'octave'/"
> on kernel.py
> 
> This makes codemirror use octave's highlighting and it works fine.

@EmericT I tried copying in "s/'name': 'scilab'/'name': 'octave'/" to my kernel.py but it didn't work. This is what I have now:

```
class ScilabKernel(ProcessMetaKernel):
    implementation = 'Scilab Kernel'
    implementation_version = __version__,
    language = 'scilab'
    language_version = __version__,
    banner = "Scilab Kernel",
    language_info = {
        'name': 'scilab',
        'file_extension': '.sci',
		"mimetype": "text/x-octave",
        "s/'name': 'scilab'/'name': 'octave'/"
        "version": __version__,
        'help_links': MetaKernel.help_links,
    }
    kernel_json = get_kernel_json()
```

Any advice?

**Comment by EmericT** [comment]
I've not been very clear indeed : the line 's/(...)/(...)' was meant to express a substitution in the original file. 

Try to replace the line 
'name': 'scilab' 
with
'name': 'octave' 
(and remove the line "s/'name': 'scilab'/'name': 'octave'/" )

**Comment by blink1073** [comment]
I had expected the mimetype to take care of that.  I'll make a change to the name.

**Comment by blink1073** [comment]
Released 0.9.6.
